src/Hydrators/CatHydrator.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace TopCat\Hydrators;
+
+use TopCat\Entities\CatEntity;
+
+class CatHydrator
+{
+    private $db;
+
+    public function __construct(\PDO $db)
+    {
+        $this->db = $db;
+    }
+
+    /**
+     * 
+     * @param int - Takes user input in the form of $breedID on page load following selection of breed from dropdown
+     * 
+     * @return array - Returns an array of Cat objects based on details acquired from database
+     */
+    public function createCatEntitiesArray(\PDO $db, int $breedID): array
```

Comment:
why is $db being passed in here? You're already passing it in in the constructor

Reply:
doing